src/policy/packages.h

```diff
@@ -49,13 +51,36 @@ using Package = std::vector<CTransactionRef>;
 
 class PackageValidationState : public ValidationState<PackageValidationResult> {};
 
+/** If any direct dependencies exist between transactions (i.e. a child spending the output of a
+ * parent), checks that all parents appear somewhere in the list before their respective children.
+ * No other ordering is enforced. This function cannot detect indirect dependencies (e.g. a
+ * transaction's grandparent if its parent is not present).
+ * @returns true if sorted. False if any tx spends the output of a tx that appears later in txns.
+ */
+bool IsSorted(const Package& txns);
```

Comment:
nit: `IsSorted` does not convey what is being checked for sortedness.

```suggestion
bool IsPackageSorted(const Package& txns);
```

Reply:
Ok, renamed this to `IsTopoSortedPackage`, i.e. "is a topologically sorted package" to be more descriptive.

Reply:
can you do the same for `IsConsistent`? e.g. `IsPackageConsistent`

Reply:
Renamed to `IsConsistentPackage`

Reply:
Could alternatively make these member functions of `Package` ?